lib/redux-router-engine.js

```diff
@@ -79,26 +84,52 @@ class ReduxRouterEngine {
 
   _handleRender(req, match, options) {
     const withIds = options.withIds !== undefined ? options.withIds : this.options.withIds;
+    const profileRenderingTime = options.profileRenderingTime !== undefined ?
+      options.profileRenderingTime :
+      this.options.profileRenderingTime;
+    const filterState = this.options.filterState && typeof this.options.filterState === "function" ?
+      this.options.filterState : null;
+
     const stringifyPreloadedState =
       options.stringifyPreloadedState || this.options.stringifyPreloadedState;
 
     return (options.createReduxStore || this.options.createReduxStore).call(this, req, match)
       .then((store) => {
+
+        const startTime = profileRenderingTime && process.hrtime();
+        const html = this._renderToString(req, store, match, withIds, startTime);
+
+        if (profileRenderingTime) {
+          const endTime = process.hrtime(startTime);
+          const renderingTime = endTime[0] * THOUSAND + endTime[1] / MILLION;
+          req.log(
```

Comment:
we can't have this here for obvious reasons.  The approach to this would be to supply custom `_renderToString` callback which then can do custom measurement and logging.